tests/php/BaseSubsiteTest.php

```diff
@@ -1,16 +1,26 @@
 <?php
+
+namespace SilverStripe\Subsites\Tests;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Subsites\Model\Subsite;
+
 class BaseSubsiteTest extends SapphireTest
 {
+
     public function setUp()
     {
         parent::setUp();
 
         Subsite::$use_session_subsiteid = true;
-		Subsite::$force_subsite = null;
+        Subsite::$force_subsite = null;
     }
 
     /**
      * Avoid subsites filtering on fixture fetching.
+     * @param string $class
+     * @param string $id
+     * @return \SilverStripe\ORM\DataObject
      */
     public function objFromFixture($class, $id)
```

Comment:
Is protected now, also update the args to match SapphireTest, so `$className, $identifier`